nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -772,23 +773,24 @@ define([
 
         // bulkImportCell / cellTabs / buildTabButtons
         function buildRunControlPanelDisplayButtons(events) {
+            const cssBaseClass = 'kb-rcp';
             const buttons = Object.keys(controlBarTabs.tabs)
+                .filter((key) => {
```

Comment:
Changed this so it mirrors what is in the cellControlPanel widget. At some point this control panel and action stuff can be removed and replaced by the `cellTabs` and `cellControlPanel` widgets; there's just some extra stuff about tab selection that would need to be checked first. Oh, and tests written, of course!